Data/TypeDocumentation.cs

```diff
@@ -188,6 +188,14 @@ void XmlToHtml (XElement x, TextWriter w)
                         if (cref != null && cref.Value.Length > 2) {
                             WriteMemberLinkHtml (cref.Value, w);
                         }
+                        else {
+                            var langword = x.Attribute ("langword");
+                            if (langword != null && !string.IsNullOrWhiteSpace (langword.Value)) {
+                                w.Write ("<span class=\"inline-code c-kw\">");
+                                w.Write (langword.Value);
```

Comment:
Please use `WriteEncodedHtml(langword.Value, w)"" to make sure the string gets encoded.

Reply:
Done.